server/rest/src/main/java/org/infinispan/rest/server/operations/mediatypes/printers/Escaper.java

```diff
@@ -0,0 +1,44 @@
+package org.infinispan.rest.server.operations.mediatypes.printers;
+
+public class Escaper {
+
+   private Escaper() {
+
+   }
+
+   static String escapeHtml(String html) {
+      return escapeXml(html);
+   }
+
+   static String escapeXml(String xml) {
```

Comment:
@alanfx Nope, URl-escaping has completely different rules: `<a href` becomes `%3Ca+href` instead of `&lt;a href`.